packages/http/src/util.js

```diff
@@ -64,10 +70,14 @@ export function request(method, path, params, callback = s => s) {
       params
     );
 
-    let { body, headers = { 'Content-Type': 'application/json' } } = resolvedParams;
+    let { body, contentType = 'json', headers = {} } = resolvedParams;
 
-    if (resolvedParams.form) {
+    if (resolvedParams.headers) {
```

Comment:
I don't think this logic is right - but you should be able to prove that to me in your tests

But If the user passes in headers, the content-type will never be set

Reply:
@josephjclark  I have changed the logic